day04/index.js

```diff
@@ -0,0 +1,10 @@
+const { passwordFinder1 } = require('./passwordFinder1');
+const { passwordFinder2 } = require('./passwordFinder2');
+
+// Retrieve acceptable password options by calling either methods with a start and end value.
+
+// getPasswordsFilter filters through a generated array of all possibilites within range.
+const getPasswordsFilter = passwordFinder1;
```

Comment:
so instead of doing this, you can name the import at the top whatever you want, so you could name it getPasswordsFilter up there!

Reply:
👍  thanks!!